keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/artifacts/BuildMetadata.kt

```diff
@@ -13,7 +17,23 @@ data class BuildMetadata(
   val startedTs: String? = null,
   val completedTs: String? = null,
   val status: String? = null
-)
+) {
+  fun getStartedAtInstant(): Instant? =
+    startedAt
+      ?.let {
+        LocalDateTime.parse(it.removeSuffix("Z"))
+          .atZone(UTC)
+          .toInstant()
```

Comment:
Pretty sure you can just parse the entire string as a `ZonedDateTime` rather than having to manipulate it like that.